x/bank/module.go

```diff
@@ -170,7 +170,7 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 }
 
 // ConsensusVersion implements AppModule/ConsensusVersion.
-func (AppModule) ConsensusVersion() uint64 { return 4 }
+func (AppModule) ConsensusVersion() uint64 { return 5 }
```

Comment:
```suggestion
func (AppModule) ConsensusVersion() uint64 { return 4 }
```

No need to bump the version as it is v3 on v0.46 (https://github.com/cosmos/cosmos-sdk/blob/v0.46.0-rc3/x/bank/module.go#L160-L161). It has already been bumped in #11977